### ISSUE TITLE
vnet v6 - v1 with instance normalization

### ISSUE THREAD

**Issue by xiedidan** [xiedidan/luna-network#28]
I found it's hard to train v1 in [Issue #27](https://github.com/xiedidan/luna-network/issues/27).
With instance normalization, network should be easier to converge.

**Comment by xiedidan** [comment]
MVN does not work with 3D blobs.

**Comment by xiedidan** [comment]
switching to batch norm

**Comment by xiedidan** [comment]
v6 is training now
bn does affect the speed, reducing 3 iters/s (of v1) to 2 iters/s (of v6)
also, bn needs much more memory so it's hard to test along with training
working on memory problem now

**Comment by xiedidan** [comment]
reduced batch size to 2
all working
